input/fsh/au-erequesting-task.fsh

```diff
@@ -0,0 +1,46 @@
+Profile: AUeRequestingTask
+Parent: Task
+Id: au-erequesting-task
+Title: "AU eRequesting Task"
+Description: "This profile sets minimum expectations for a Task resource that is used to record, search, and fetch information about a task to be performed. It is based on the [Task](https://www.hl7.org/fhir/R4/task.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the Task when conforming to this profile."
+
+* ^status = #draft
+* ^extension[http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm].valueInteger = 0
+
+* groupIdentifier 1..1 MS
+* groupIdentifier ^type.profile = $AULocalOrderIdentifier
+* groupIdentifier.type 
+  * coding 1..1    
+  * coding = $v2-0203#PGN
+
+
+* status MS
```

Comment:
https://jira.hl7.org/browse/FHIR-47081 requires a ValueSet to support requirements 
Codes: all TaskStatus values except 'draft,' 'ready,' and 'entered-in-error'
( requested | received | accepted | rejected |cancelled | in-progress | on-hold | failed | completed )

Reply:
noted already to be followed up with a terminology update